src/components/FloatingDropdown/FloatingDropdownBase.st.css

```diff
@@ -68,7 +68,6 @@
 }
 
 .textContent {
-    max-width: calc(value(maxWidth) - value(iconWith));
```

Comment:
So it seems by removing this, we're breaking the ellipsis overflow of the component.

So I suggest two things:
1. let's add a test for the component with very long text
2. instead of removing this line, maybe we should change it to:
```css
    max-width: calc(100% - value(iconWidth));
```